### PR TITLE
Introduce inline records for abstract_format

### LINKED ISSUE #62: Use anonymous records for all absforms
e.g. `| ModDecl of form_t list` -> `| ModDecl of {forms: form_t list}`

### PR DESCRIPTION
**Pull request by youxkei**
This PR resolves #62.

### CODE REVIEW

**Comment by yutopp** [comment]
Thank you for your pull request. I'll check it later...

**Comment by yutopp** [comment]
This PR has large changes for unit tests, thus it will be better that there are 2 checks. Could you review this PR please, @amutake ?

File: test/test_bigint.ml [hunk 1]
```diff
@@ -6,35 +6,52 @@ let%expect_test "test_bigint.beam" =
...
         ModDecl (
-          (AttrFile 1 test_bigint.erl 1)
-          (AttrMod 1 test_bigint)
-          (AttrExport 3 (
-            (s 0)
-            (l 0)))
+          (AttrFile
+            (line      1)
+            (file      test_bigint.erl)
+            (file_line 1))
```
Comment (amutake) [inline test/test_bigint.ml]: It is very nice to become to see field names!

File: lib/abstract_format.ml [hunk 2]
```diff
@@ -505,55 +509,56 @@ and expr_of_sf sf : (expr_t, err_t) Result.t =
...
-     ExprRemoteCall (line_c, line_r, m, f, args) |> return
+     ExprRemoteCall {line; line_remote; module_expr; function_expr; args} |> return
 
   (* a function call (local) *)
-  | Sf.Tuple (4, [Sf.Atom "call"; Sf.Integer line; sf_e; Sf.List sf_es]) ->
-     let%bind e = sf_e |> expr_of_sf |> track ~loc:[%here] in
-     let%bind es = sf_es |> List.map ~f:expr_of_sf |> Result.all |> track ~loc:[%here] in
-     ExprLocalCall (line, e, es) |> return
+  | Sf.Tuple (4, [Sf.Atom "call"; Sf.Integer line; sf_function_expr; Sf.List
+  sf_args]) ->
```
Comment (amutake) [inline lib/abstract_format.ml]: nit: unnecessary newline

**Comment by amutake** [comment]
Thanks!
I will merge this PR after the CI is passed.

**Pull request merged**